config/webpack.config.dev.js

```diff
@@ -196,39 +196,51 @@ module.exports = {
     ]
   },
   plugins: [
-    new CopyWebpackPlugin([{
-      from: 'src/ovirt-web-ui.config',
-    }, {
-      from: 'branding',
-      to: 'branding',
-      toType: 'dir'
-    }]),
+    new CopyWebpackPlugin([
+      {
+        from: 'src/ovirt-web-ui.config',
+      },
+      {
+        from: 'branding',
+        to: 'branding',
+        toType: 'dir'
+      }
+    ]),
+
     // Generates an `index.html` file with the <script> injected.
     new HtmlWebpackPlugin({
+      filename: 'index.html',
       inject: true,
-      template: paths.appHtml,
+      template: `!!handlebars!${paths.appHtml}`,
+      jspSSO: false,
     }),
+
     // Makes some environment variables available to the JS code, for example:
     // if (process.env.NODE_ENV === 'development') { ... }. See `env.js`.
     new webpack.DefinePlugin(env),
+
     // This is necessary to emit hot updates (currently CSS only):
     new webpack.HotModuleReplacementPlugin(),
+
     // Watcher doesn't work well if you mistype casing in a path so we use
     // a plugin that prints an error when you attempt to do this.
     // See https://github.com/facebookincubator/create-react-app/issues/240
     new CaseSensitivePathsPlugin(),
+
     // If you require a missing module and then `npm install` it, you still have
     // to restart the development server for Webpack to discover it. This plugin
     // makes the discovery automatic so you don't have to restart.
     // See https://github.com/facebookincubator/create-react-app/issues/186
     new WatchMissingNodeModulesPlugin(paths.appNodeModules),
 
     //global jquery is provided to any webpack modules
-    /*new webpack.ProvidePlugin({
+    /*
+    new webpack.ProvidePlugin({
```

Comment:
Is this still relevant, I think we can remove this comment.

Reply:
Done.